src/content/docs/style-guide/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -7,66 +7,102 @@ topics:
+4. You will be prompted to name your new branch. Descriptive names are best. It's a great way to quickly clue people in to what your work is all about. For example, if you are working on What’s New pages, you might name the branch **Whats-new-updates**.
+
+When you create a new branch, don't forget to add the Jira issue's key (DOC-1234) to the branch name and the PR title. 
+
+## Set up your local environment [#set-local]
 
 Install GitHub Desktop, and then navigate to GitHub Desktop's preferences. On Macs, click on **GitHub Desktop** in the top left corner of your screen and select **Preferences**. Select the blue **Sign In** button and follow the prompts in the browser window. 
 
-Once GitHub Desktop is set up, navigate to the [Docs Site repository](https://github.com/newrelic/docs-website) on GitHub. Click the green **Code** button and then select **Open with [GitHub Desktop](https://desktop.github.com/)**. Choose the location where you the repo, and this will clone the entire repository to your local machine at the designated path. You can ensure the repo was cloned by navigating to your local GitHub folder (The default is `~/Documents/github`). Once you have cloned the repo, you don't need to clone it again in the future.  
+Once GitHub Desktop is set up, navigate to the [Docs Site repository](https://github.com/newrelic/docs-website) on GitHub. Click the green **Code** button and then select **Open with [GitHub Desktop](https://desktop.github.com/)**. Choose the location where you want the repo, and this will clone the entire repository to your local machine at the designated path. You can ensure the repo was cloned by navigating to your local GitHub folder (the default is `~/Documents/github`). Once you have cloned the repo, you don't need to clone it again in the future.  
 
-## Run the site locally
+## Run the site locally [#run-site]
 
-Build the site locally using the Terminal to preview changes before opening a Pull Request. While it's **highly** recommended to build the site locally, this is technically an optional step. The site will automatically reflect any local changes once build. Node and Yarn are tools used to build the site on your local machine.
+Build the site locally using the terminal to preview changes before opening a Pull Request. While it's **highly** recommended to build the site locally, this is technically an optional step. The site will automatically reflect any local changes once build. Node and Yarn are tools used to build the site on your local machine.
 
-### Prerequisites
+### Prerequisites [#prereq]
 - [Install Node](https://nodejs.org/en/download/)
 - Install Yarn 
   - `npm install -g yarn`
 
-### Build the site
-1. In your Terminal, go to your cloned repo, **docs-website**. 
+### Build the site [#build]
+1. In your terminal, go to your cloned repo, **docs-website**. 
     - `cd ~/Documents/github/docs-website`
 2. Run yarn with the following commands:
     - `yarn`
     - `yarn start`
```

Comment:
Can’t we say to just run ‘yarn && yarn start’?

Reply:
Good catch!